frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -236,54 +245,69 @@ export const EditZing = () => {
       />
 
       <Box
-        display="flex"
-        flex-direction="row"
-        align-items=" center"
-        justifyContent="space-between"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          gap: 3,
+          height: (theme) => theme.spacing(17),
+          p: 6,
+          pb: 4,
+          borderBottom: 1,
+          borderBottomColor: 'essentials.12',
+        }}
       >
-        <StyledLogoWrapper>
-          <StyledLogo />
-          <StyledText>{courseInfo.names.join(', ')}</StyledText>
-        </StyledLogoWrapper>
-        s
-        <Button sx={{ height: '40px', mt: '10px' }}>
-          {selectedGroups.length === 0 ? 'Send Email To' : 'Email Selected'}
+        <LscIcon sx={{ height: '50px', width: '50px' }} />
+        <Typography variant="h4" component="h1">
+          {courseInfo.names.join(', ')}
+        </Typography>
+        <Box flexGrow={2} />
+        <Button>
+          {selectedGroupNumbers.length === 0
+            ? 'Send Email To'
+            : 'Email Selected'}
         </Button>
       </Box>
-      <DndProvider backend={HTML5Backend}>
-        <Grid container spacing={1}>
-          <UnmatchedGrid
-            unmatchedStudents={unmatchedStudents}
-            moveStudent={moveStudent}
-            matchStudents={matchStudents}
-          />
-          {studentGroups.map((studentGroup, index) => (
-            <GroupGrid
-              key={index}
-              studentList={studentGroup.memberData}
-              groupNumber={studentGroup.groupNumber}
-              shareMatchEmailTimestamp={studentGroup.shareMatchEmailTimestamp}
+
+      <Box m={6}>
+        <DndProvider backend={HTML5Backend}>
+          <Grid container spacing={1} padding={0}>
+            <UnmatchedGrid
+              unmatchedStudents={unmatchedStudents}
               moveStudent={moveStudent}
-              createTime={studentGroup.createTime}
-              updateTime={studentGroup.updateTime}
-              selected={selectedGroups.some(
-                (g) => g.groupNumber === studentGroup.groupNumber
-              )}
-              handleChecked={(event) => {
-                editSelectedGroups(studentGroup, event.target.checked)
-              }}
+              matchStudents={matchStudents}
             />
-          ))}
-        </Grid>
-      </DndProvider>
-    </StyledContainer>
+            {studentGroups.map((studentGroup, index) => (
+              <GroupGrid
+                key={index}
```

Comment:
I think key can be tied to `studentGroup.groupNumber` since it's a unique identifier anyway as opposed to just index.